aggregator/src/aggregator/test_util.rs

```diff
@@ -200,3 +203,15 @@ pub async fn assert_task_aggregation_counter(
         sleep(Duration::from_millis(100)).await;
     }
 }
+
+/// Extension methods on [`Clock`].
+pub trait ClockExt: Sized {
+    fn now_at_batch_interval_start(&self, time_precision: &janusDuration) -> Time;
```

Comment:
I find the name of this method confusing, but I think that's due in part to the underlying `to_batch_interval_start` being badly named, too. That method is documented as "Compute the start of the batch interval containing this Time, given the task time precision" but I don't think that's right: what it does is round the time down to the provided precision. We must have changed what it does along the way without renaming it.

In any case, I think we could name this `ClockExt` method better. Is `ClockExt::now_aligned_to_precision` correct in your view?

Reply:
I think that's a great name.

Reply:
Updated in 693c69b2868b38242eca1584aa5c1e2ec56004d4